listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -428,8 +431,8 @@ def fetch_recent_listens_for_users(self, user_list, limit=2, max_age=3600):
                 result = curs.fetchone()
                 if not result:
                     break
-
-                listens.append(Listen.from_timescale(*result[0:8]))
+                user_name = user_id_map[result["user_id"]]
+                listens.append(Listen.from_timescale(*result[0:8], user_name=user_name))
```

Comment:
it'd be great to update calls to `from_timescale` to use the dict format from `result`, and explicitly list them all. This way we won't have unexpected errors if we happen to change the order of the columns in the select.

Reply:
How about `**result`?

Reply:
this would require the column names and parameter names in from_timescale to be the same, which might not be the case. I think that explicitly listing them might be safest.

Reply:
Makes sense, fixed.